LayoutTests/imported/w3c/resources/import-expectations.json

```
@@ -304,6 +304,9 @@
     "web-platform-tests/html/interaction/focus/the-autofocus-attribute": "import",
     "web-platform-tests/html/obsolete": "skip",
     "web-platform-tests/html/rendering": "skip",
+    "web-platform-tests/html/rendering/non-replaced-elements/tables": "import",
+    "web-platform-tests/html/rendering/non-replaced-elements/tables/table-bordercolor-001-ref.html": "import",
+    "web-platform-tests/html/rendering/non-replaced-elements/tables/table-bordercolor-001.html": "import",
```

can you avoid adding unnecessary paths?

---

Could you explain? :) I'm not sure what is the goal of this file here. 
This was created probably when I reset the tests.

---

```suggestion
    "web-platform-tests/html/rendering/non-replaced-elements/tables": "import",
```
This is what I'm suggesting, the 2 other paths are included in the first one.